src/coreclr/vm/method.hpp

```diff
@@ -1710,6 +1696,20 @@ class MethodDesc
     WORD m_wFlags; // See MethodDescFlags
     PTR_MethodDescCodeData m_codeData;
 
+#ifdef _DEBUG
```

Comment:
Interesting. Can we do the similar move for `MethodTable` to simplify managed definition? Is there any external diagnostic tool depending on the layout without going through DAC code in this repo?